### DISCUSSION TITLE
FreeStyle Libre 2 incorrect readings

### DISCUSSION THREAD

**Discussion by wariw** [NightscoutFoundation/xDrip#2962] [Q&A]
Hello,

My fiancée has been using xDrip with FreeStyle Fibre for almost two years now (first Libre 1 with MiaoMiao than Libre 2 since the beginning of 2023). Her current setup consists of: xDrip on Xiaomi phone, Libre 2 and Watchdrip+ with Mi Band 4. Everything was working fine until 2-3 months ago when we moved from stable release to nightly because of https://github.com/NightscoutFoundation/xDrip/issues/2725 . From that point onward we observed very strange behavior, readings from LibreLink were sometimes massively incorrect compared to glucose meter readings (usually much lower, event when sugar level was stable over long period of time). After uninstalling xDrip, readings in LibreLink corrected itself over time. I am not 100% sure if xDrip was the cause, but similar scenario happened multiple times so far.

I would like to know, is it possible that xDrip (or Watchdrip+?) actually affects readings from Libre 2 in LibreLink? Has anyone had similar problems?

PS. Another lead in that case was provided by Abbot support, they were aware of problems like this caused by smartwatches after using many sensors (supposedly caused by problems with bluetooth and historic sensor connections), I am not sure how this relates to all of this.

**Comment by PhilTU** [comment]
Librelink should not be affected by anything you do. It should collect data from the sensor, process it and display results.

xDrip+ can collect the same raw data, process it, display it, pass it to watchdrip+ which then passes it to your Miband.

These data flows are one way, from the sensor to your eyes.

It may be that the new sensor type (which led you to change versions) has a different behaviour with Librelink. But I do not see how the glucose value can be disrupted. Best practice is to disable or remove Librelink, to avoid conflict of communication.

**Reply by wariw** [reply]
I suspected this as well, I can't really imagine scenario where reading operations actually affect sensor measurement, but information about smartwatches affecting sensor and causing incorrect reading provided by Abbott tech support is strange. I have tried to monitor this problems more, but there are too many factors affecting sugar levels to draw any conclusions.

We had very similar problem two sensors ago. Over time sensor started to report more and more low values that were not confirmed by glucose meter. On the eight day sensor stopped working altogether and was replaced by Abbott. Sensor that was sent as replacement worked fine so I have assumed that it was just faulty sensor, but current sensor have the same problem.

This might just be a coincidence and faulty sensors, but it all started when sensors with new serial numbers started to be shipped. I will monitor this situation more.

As for uninstalling LibreLink, unfortunately most physicians in Poland use is for remote monitoring so we must use it alongside xDrip+.
